terraform/aws/implementation/modules/eks/data.tf

```diff
@@ -351,3 +351,7 @@ data "kubernetes_ingress_v1" "ingress" {
     }
   }
 }
+
+data "aws_ecrpublic_authorization_token" "token" {
```

Comment:
Glad this is already set to `us-east-1`.  I think there was a bug reported because the public ECR is defaulted to Virginia for the `aws_ecrpublic_authorization_token`.